test/isArgument_test.js

```diff
@@ -0,0 +1,47 @@
+goog.require('helper');
```

Comment:
Important note about some of the tests in this file.  While it looks like they tests pass as written, the use of arrow functions means that you're making different assertions than you might think.  I would rewrite this to avoid use of the arguments keyword inside arrow functions.

As for why:
Arrow functions [do not have a binding for arguments](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

These tests all pass because the test methods themselves are standard functions which do have an arguments binding, and the arrow functions have closure over that.

For a concrete example:
```
function foo() {
  const bar = () => {
    console.log(`bar: ${[].slice.apply(arguments)}`);
  };
  console.log(`foo: ${[].slice.apply(arguments)}`);
  bar('some other argument');
}

foo('The original argument');
// Output:
// foo: The original argument.
// bar: The original argument.
```

Reply:
I did not know this, changing now. Thank you!